package.json

```diff
@@ -27,9 +27,10 @@
         "unlink-runtime": "rm -rf node_modules/@nmshd && npm install",
         "tsc": "tsc",
         "tsc:watch": "tsc -w",
-        "test:ci": "npm run build && jest -i",
+        "test:ci": "npm run build && npx c8 jest -i",
```

Comment:
we don't use npx in scripts. npm is resolving executables already internally. same for the other script.